src/main.cpp

```diff
@@ -4174,6 +4174,8 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
 
         LOCK(cs_main);
 
+        if (IsInitialBlockDownload()) return true;
```

Comment:
Yes, it's completely inconsequential, but pedantically the `return true` should be on the next line and indented.